docs/flow/modin/engines/base/series.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
```

Comment:
Since we removed stuff related to internal Series object this should be removed as well.

Reply:
Fixed.